### PR TITLE
Fileno error fix

### PR DESCRIPTION
**Pull request by useHTML5**
Postgres error after https://github.com/django-dbbackup/django-dbbackup/commit/8d74de2dd5457f866784813f2edb36085c817a8d#diff-d2dc461d08a22c2ab789df9bbd5e3403

### CODE REVIEW

**Comment by ZuluPro** [comment]
Hello @useHTML5 
Could you write some tests ensuring this PR ?

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 pytz
 six
 Django>=1.5
+paramiko #django-dbbackup
```
Comment (ZuluPro) [inline requirements.txt:4]: Why paramiko ?

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+django-storages #django-dbbackup
```
Comment (ZuluPro) [inline requirements.txt:5]: django-storage should be an optional dependency

**Comment by ZuluPro** [comment]
Hello @useHTML5 
Could you also merge your commits into 1 ?

**Comment by jonathan-s** [comment]
Since it's unclear what this is trying to fix I'm closing this.

**Pull request closed without merging**